cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/SparkProgramRuntimeProvider.java

```diff
@@ -94,7 +94,7 @@ public ClassLoader createProgramClassLoader(CConfiguration cConf, ProgramType ty
     boolean rewriteYarnClient = cConf.getBoolean(Constants.AppFabric.SPARK_YARN_CLIENT_REWRITE);
 
     try {
-      ClassLoader sparkRunnerClassLoader = sparkRunnerClassLoader = createClassLoader(filterScalaClasses,
+      SparkRunnerClassLoader sparkRunnerClassLoader = sparkRunnerClassLoader = createClassLoader(filterScalaClasses,
```

Comment:
I understand it was here, but could you remove this double assignment? It's confusing

Reply:
Done.